lib/tasks/reports.rake

```diff
@@ -8,6 +8,9 @@ namespace :reports do
     uploader = User.find(requester_id)
 
     abort "Report #{report.id} has not been approved" unless report.approved?
+    unless report.financial_year.present? && report.financial_quarter.present?
+      abort "Report #{report.id} has no financial year or quarter"
+    end
     abort "Report #{report.id} not found" if report.nil?
```

Comment:
Sorry for not noticing this earlier! I think the `report.nil?` check should be first.